lib/progressTracker.js

```diff
@@ -63,18 +62,32 @@ function track (instance, opts) {
 
     addedListeners = true
 
+    function cancel () {
+      if (iOpts.amount) {
+        amountProgressBar.tick(iOpts.amount - 1)
+      } else {
+        durationProgressBar.tick(iOpts.duration - 1)
+      }
+      instance.cancel()
+    }
```

Comment:
Can you export it from this module and add the SIGINT handler in the runTracker function in autocannon.js instead?

Reply:
Will do!